src/app/shell/info/about/about.component.html

```diff
@@ -1,20 +1,22 @@
-<div *ngIf="(platformInformation$ | async)?.companyInformationItems.length; else noInfo">
-  <div class="info-content">
-    <div>
-      <h2 class="info-title">{{ (platformInformation$ | async).title }}</h2>
-    </div>
-    <div class="info-text">
-      <div *ngFor="let item of (platformInformation$ | async)?.companyInformationItems">
-        <p><strong>{{ item.sectionName }}</strong></p>
-        <p>{{ item.description }}</p>
+<ng-container *ngIf="!(isLoading$ | async)">
+  <div *ngIf="(platformInformation$ | async)?.companyInformationItems.length; else noInfo">
+    <div class="info-content">
+      <div>
+        <h2 class="info-title">{{ (platformInformation$ | async).title }}</h2>
+      </div>
+      <div class="info-text">
+        <div *ngFor="let item of (platformInformation$ | async)?.companyInformationItems">
```

Comment:
let's do subscription in ts file, because we use async more than 1 time

Reply:
Done (in three components )